mailchimp-connector/apps/mailchimp.yaml

```diff
@@ -0,0 +1,743 @@
+#!content-type:audience-csv
+#!content-type:campaign
+#!content-type:list
+appClass: info.magnolia.ui.framework.app.BaseApp
+icon: icon-mail-setting
+class: info.magnolia.ui.contentapp.configuration.ContentAppDescriptor
+label: app.name
+
+subApps:
+  campaigns:
+    label: subapp.campaigns.name
```

Comment:
I'd prefer the translation key to be subapp.campaigns.label instead of subapp.campaigns.name